fhir-server/src/main/java/com/ibm/fhir/server/filter/rest/FHIRRestServletFilter.java

```diff
@@ -256,12 +255,14 @@ private HTTPHandlingPreference computeHandlingPref(ServletRequest request) throw
         return handlingPref;
     }
 
-    private HTTPReturnPreference computeReturnPref(ServletRequest request, HTTPHandlingPreference handlingPref) throws FHIRException {
+    private void computeReturnPref(FHIRRequestContext context, ServletRequest request, HTTPHandlingPreference handlingPref) throws FHIRException {
         HTTPReturnPreference returnPref = defaultHttpReturnPref;
```

Comment:
Now that we have the `setReturnPreferenceDefault` flag and different defaults based on the interaction, I wonder if it makes sense to keep this null in cases where its not explicitly passed (forcing everywhere using it to use their own "isDefault"->set returnPref logic)...

Reply:
If not, then it might be worth leaving a quick comment here to explain that this `defaultHttpReturnPref` is for the spec-defined uses but that interactions may have their own defaults

Reply:
Yeah, I'll add the comment.